src/borg/archiver.py

```diff
@@ -2733,6 +2733,8 @@ def build_parser(self):
             'a_status_oddity': '"I am seeing ‘A’ (added) status for a unchanged file!?"',
             'separate_compaction': '"Separate compaction"',
             'list_item_flags': '"Item flags"',
+            'key_files': 'Internals -> Data structures and file formats -> Key files in the online documentation',
```

Comment:
guess i would drop the " in the online documentation" part.

the reader is already reading the docs somewhere (doesn't matter whether online or not), so a reference naturally refers to the same docs.

Reply:
As far as I can tell:
1. This placeholder is _only_ printed by `borg help init`
2. The `borg help` only supports help for specific commands - only a fraction of the borg docs

In this situation I do not see a better solution than referring to the more complete online docs

Reply:
But this is nothing new, right? so it also applies to the lines above and we should stay consistent.

Reply:
I see your point. Will fix in the next commit